resources/views/recipes/show.blade.php

```diff
@@ -121,25 +122,141 @@
+                      4
+                    </label> 
+                    <label class="radio-inline" for="rating-4">
+                      <input type="radio" name="rating" id="rating-4" value="5">
+                      5
+                    </label>
+                  </div>
+
+                </div>
+                
+                <!-- Textarea -->
+                <div class="form-group">
+                  <label class="col-md-2 control-label" for="body">Bericht</label>
+                  <div class="col-md-8">                     
+                    <textarea class="form-control" id="body" name="body"></textarea>
+                  </div>
+                </div>
+                
+                <!-- Submit button -->
+                <div class="form-group">
+                  <label class="col-md-2 control-label" for="body"></label>
+                  <div class="col-md-8">                     
+                    <button type="submit" class="btn btn-success">Verstuur je reactie &raquo;</button>
+                  </div>
+                </div>
+                </fieldset>
+            </form>
+
+            @endif
+            <div class="row">
+        		                
+                <div class="list-group">
+                @if($recipe->comments->count() ==0)
+                <p class="text-center well">Geen reacties gevonden...</p>
+                @else
+                  @foreach($recipe->comments()->orderBy('id','DESC')->get() as $comment)
+                  <div class="list-group-item">
+
+                       <div class="pull-right">
+                            <small><i class="fa fa-user fa-fw"></i> {{$comment->author->name}}</small>
+                            <br>
+
+                           <div class="stars text-left">
+                               @if ($comment->rating != NULL)
+                                {!!$comment->getHtmlStars()!!}
+                               @else
+                                   Geen rating
+                               @endif
+                            </div>
+
+                        </div>
+
+                        <h4 class="list-group-item-heading">
+                            {{$comment->title}}
+                            @if (Auth::user())
+                                @if ($comment->user_id == Auth::user()->id)
+                                    <a class="btn btn-xs text-danger pull-right" href="{{route('recipes.deleteComment',[$recipe->id, $comment->id])}}"><i class="fa fa-trash-o fa-fw"></i></a>
+                                @endif
+                            @endif
+                            <br>
+                            <p style="font-size: 75% !important;"><small class="text-muted">{{$comment->created_at}}</small></p>
+
+                        </h4>
+                        <p class="list-group-item-text">
+                            {!!nl2br($comment->body)!!}
```

Comment:
Arbitrary HTML code can be injected in the comment body by anyone, because Blade's raw output tags are used here.